samples/kubernetes/deployment.yaml

```diff
@@ -1,3 +1,16 @@
+# Secret used to set up HTTP basic auth between the service broker and the service catalog.
+apiVersion: v1
+kind: Secret
+metadata:
+  name: atlas-service-broker-auth
+type: Opaque
+stringData:
+  username: username
+  password: password
```

Comment:
Just for the sake of extreme clarity, make it clear that these can be any string. I'd say do something like this:
```
stringData:
  username: my-username
  password: my-password
```

Reply:
Changed!